jobbergate-cli/jobbergate_cli/subapps/job_scripts/tools.py

```diff
@@ -153,7 +153,7 @@ def get_template_output_name_mapping(config: JobbergateConfig) -> Dict[str, str]
             log_message="Entry point file not specified",
         )
 
-    output_name_mapping = {config.default_template: output_dir / remove_prefix_suffix(config.default_template)}
+    output_name_mapping = {config.default_template: output_dir / f"{job_name}.job"}
```

Comment:
Can we make this behavior optional (maybe activate with an environment variable or something). It's very non-intuitive to rename a python or bash script to end with a ".job" extension. This is specific to one customer, and I think it's actually worse behavior for the project.

Reply:
Good point. The new setting `JOBBERGATE_LEGACY_NAME_CONVENTION` was added to control it.